modules/userId/eids.js

```diff
@@ -207,7 +207,10 @@ const USER_IDS_CONFIG = {
       return data.id;
     }
   },
-
+  'deepintentId': {
+    source: 'deepintent.com',
+    atype: 1
```

Comment:
if this is an email-based identifier, then it should be `3` here I think. @patmmccann you've been reviewing a lot of these, do you agree?

Reply:
I will make the change, Is there a list as well as which is being referred ?

Reply:
https://github.com/InteractiveAdvertisingBureau/AdCOM/blob/master/AdCOM%20v1.0%20FINAL.md#list_agenttypes